docs/en/src/develop/developer.md

```diff
@@ -0,0 +1,3 @@
+# For Developer
+
+comming soon...
```

Comment:
It might be nice to have an issue with a rough list of what you want to write in this section.